lib/Sema/TypeCheckType.cpp

```diff
@@ -2679,6 +2683,7 @@ Type TypeResolver::resolveOpaqueReturnType(TypeRepr *repr,
   auto ty = builder.resolveOpaqueType(opaqueNode, TypeArgs, ordinal);
```

Comment:
Same question here.

Reply:
If any of the `TypeArgs` contain ErrorType, then the resulting opaque type would transitively contain one.